src/hardware/adlib.cpp

```diff
@@ -599,8 +602,8 @@ Bitu Module::CtrlRead( void ) {
 	return 0xff;
 }
 
-
-void Module::PortWrite( Bitu port, Bitu val, Bitu /*iolen*/ ) {
+void Module::PortWrite(Bitu port, uint8_t val, Bitu /*iolen*/)
```

Comment:
Note for reviewers: The Adlib and OPL IO ports read (return) and write (accept) 8-bit.